app/assets/stylesheets/home.scss

```diff
@@ -1,6 +1,6 @@
 .masthead-video {
   background-color: black;
-  position: fixed;
+  position: absolute;
   top: 50%;
```

Comment:
This change might be causing the appearance of a pannable area on the right when browser window width is smaller than video width.